bazel_integration_test/bzlmod/bazel_binaries.bzl

```diff
@@ -118,14 +123,26 @@ version, version_file.\
         _bazel_binary_repo_rule(
             name = vi.repo_name,
             version_file = download.version_file,
+            bazelisk = "@%s//:bazelisk" % bazelisk_repo_name,
         )
     return vi
 
+# TODO: Make this configurable.
+_BAZELISK_VERSION = "1.18.0"
```

Comment:
I noticed that this version value is also hard-coded in the `bazel_binaries` macro. Do we need this value in both places?

Reply:
In the non-bzlmod version, this is an parameter to the `bazel_binaries` macro which can be overriden but defaults to 1.18.0. I couldn't figure out the syntax for bzlmod extensions (I think it needs a separate tag?), so it's just hardcoded here. Let me know if you'd prefer me to figure that out.

Reply:
Could we create an issue and mark this with a `GHXXX` comment? I think that would be sufficient for now.

Reply:
Done, filed #184.